Controllers/JobOffer/JobOfferController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using vabalas_api.Controllers.Job.Dtos;
+using vabalas_api.Controllers.JobOffer.Dtos;
+using vabalas_api.Service;
+using vabalas_api.Service.Impl;
+using vabalas_api.Models;
+
+namespace vabalas_api.Controllers.JobOffer
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class JobOfferController : ControllerBase
+    {
+        private readonly IJobOfferService _jobOfferService;
+
+        public JobOfferController(IJobOfferService jobOfferService)
+        {
+            _jobOfferService = jobOfferService;
+        }
+        [HttpGet]
+        public async Task<IActionResult> GetAll()
+        {
+            return Ok(await _jobOfferService.FindAll());
+        }
+        [HttpGet("offer/{offerId}")]
```

Comment:
does not need "offer/{offerId}"
leave "{offerId}" only
GetAll reguest trigers a GET req with no path variables